infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -482,9 +482,13 @@ public synchronized void saveManagementLayout(Layout layout) {
             dataStore.put(Layout.class, PREFIX_MANAGEMENT, MANAGEMENT_LAYOUT, layout);
             log.info("saveManagementLayout: Updated to new layout at epoch {}",
                     getManagementLayout().getEpoch());
-        } else {
+        } else if (layout.getEpoch() < currentLayout.getEpoch()){
             log.debug("saveManagementLayout: "
-                            + "Ignoring layout because new epoch {} <= old epoch {}",
+                            + "Ignoring layout because new epoch {} < old epoch {}",
```

Comment:
I think the "Ignoring layout ...." statement can be trace, but we should have an info level log about anytime we accept a newer layout.

Reply:
Done